src/lib/db/clients/postgresql.js

```diff
@@ -486,6 +487,34 @@ export async function updateValues(conn, updates) {
   return results
 }
 
+export async function deleteRows(conn, updates) {
+  const deleteQueries = updates.map(update => {
+    let where = {}
+    where[update.pkColumn] = update.primaryKey
+    return knex(update.table)
```

Comment:
can we use Knex for mysql also?

I think it's much nicer using knex vs manually building queries (which always makes me nervous no matter how often I do it)